packages/customize-widgets/src/filters/widget-box-parent-support.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+
+import { addFilter } from '@wordpress/hooks';
+
+/**
+ * Removes the "parent" property from the Widget Box block
+ * so as to allow it to appear in the Widgets Customizer.
+ * By default this block is set to appear only as a child
+ * of "widget-area" so as to avoid nesting of Widget Boxes.
+ * However as the Customizer doesn't utilise the `widget-area`
+ * block, we must remove the parent setting.
+ *
+ * @param {Object} settings the block settings.
+ * @param {string} name     the name of the block.
+ * @return {Object} the new block settings.
+ */
+function removeWidgetBoxParent( settings, name ) {
+	if ( name !== 'core/widget-box' ) {
+		return settings;
+	}
+
+	// Remove the "parent" property.
+	// eslint-disable-next-line no-unused-vars
+	const { parent, ...newSettings } = settings;
+
+	return newSettings;
+}
+
+addFilter(
```

Comment:
It's just my preference, but I don't like using hooks in JavaScript 😅, it makes it very hard to trace code. Instead, could we just override the settings when we are registering the block type? It matches how the other registrations work in both editors too.

Reply:
> could we just override the settings when we are registering the block type?

Ah I see. Pass some arguments into the `registerWidgetBoxBlock()` method to choose how to configure it? Seems sensible. Happy to avoid filters!

Reply:
This restriction is now removed entirely on @tellthemachines's recommendation.